pkg/sql/sqlbase/structured.proto

```diff
@@ -664,6 +737,27 @@ message TableDescriptor {
   // hlc timestamp to ensure that this field is set properly when extracted from
   // a Descriptor.
   optional util.hlc.Timestamp modification_time = 7 [(gogoproto.nullable) = false];
+  // A list of draining names. The draining name entries are drained from
+  // the cluster wide name caches by incrementing the version for this
+  // descriptor and ensuring that there are no leases on prior
+  // versions of the descriptor. This field is then cleared and the version
+  // of the descriptor incremented.
+  repeated NameInfo draining_names = 21 [(gogoproto.nullable) = false];
```

Comment:
Is it safe/backwards compatible to move a message out of another message like this?

Reply:
Yes, protos are defined as just their set of fields IIUC. The nesting is really just a naming thing for the generated code and for interacting with imports.